### ISSUE TITLE
Incorrect certificate path is given on Windows

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#8315]
See https://community.letsencrypt.org/t/created-certificate-but-it-isnt-in-directory/134076/.

The TL;DR is Certbot told the user the certificate was under `C:\Windows\system32` but they found it at `c:\Windows\syswow64`. This is almost certainly due to the behavior described at https://community.letsencrypt.org/t/created-certificate-but-it-isnt-in-directory/134076/11 and the fact that when you run Certbot with --csr it defaults to writing the certificate to the current working directory of your shell.

This is an edge case, but is there anything sane we can do here? Special casing this particular output doesn't seem great, but maybe it's worth doing or maybe there's a better approach.

**Comment by adferrand** [comment]
Yeah I was a little worried that we could encounter some edge cases by exposing only a 32bits version of Certbot for Windows. Except some branching in the processing that does not worth the complexity for this case in my opinion, we could take a leap and provide the 64bits version only, or both with a strong emphasis on the 64bits one.

Indeed almost nobody uses Windows 32bits nowadays, and Microsoft also dropped its support in the last Windows 10 update (see https://www.bleepingcomputer.com/news/microsoft/rip-microsoft-to-drop-support-for-windows-10-on-32-bit-systems/).

**Comment by bmw** [comment]
If the vast majority of our Windows users could get away with us only having 64-bit support, I'd be tempted to do that. Unfortunately, I don't think we can currently get data on how many users we have running 32-bit Windows, but maybe we can make the change without having exact data on this.

If we're going to make a change here, I think we may want to do so before releasing the auto-update mechanism since adding or removing architectures could complicate things. I added a comment to that effect in the PR.

**Comment by adferrand** [comment]
According to the Steam hardware suvey of August 2021, 99.64% of Windows users on Steam are using a 64bit flavor of the OS.

https://store.steampowered.com/hwsurvey/Steam-Hardware-Software-Survey-Welcome-to-Steam
